desktop/apps/partnerships/client/router.coffee

```diff
@@ -29,3 +30,6 @@ module.exports = class PartnershipsRouter extends Backbone.Router
     $nav = $ '.partnerships-section-nav'
     selector = "##{slug}"
     @jump.scrollToPosition $(selector)?.offset()?.top - $nav.outerHeight() / 2
+
+  toApply: ->
+    @toSection('apply')
```

Comment:
I'm unsure if this "Apply" button ever worked like the other quick-nav items as its structured differently. This seemed to be the most direct fix.